core/expression/src/vm/vm.rs

```
@@ -833,7 +833,7 @@ impl<'arena, 'parent_ref, 'bytecode_ref> VMInner<'parent_ref, 'bytecode_ref> {
                     let a = self.pop()?;
 
                     match (a, b) {
-                        (Number(a), Number(b)) => self.push(Number(a / b)),
+                        (Number(a), Number(b)) => self.push(a.checked_div(b).map_or(Null, Number)),
```

We should throw an opcode error in this case, division by zero shouldn't lead to a "Null"

---

Updated! We can generalize the error to be for all undefined numeric operations (e.g. `x % 0`), though for this issue I'm keeping it to just division.